config/initializers/paperclip.rb

```diff
@@ -66,6 +66,8 @@ module Paperclip
   # Checks file for spoofing
   class MediaTypeSpoofDetector
     def spoofed?
+      return false if ENV['SPOOFING_CHECK_DISABLED'] && ENV['RUNNING_LOCALY']
```

Comment:
Does this "just" work? In any case, I would just change the variable to be called `DISABLE_SPOOF_CHECKING`.